### DISCUSSION TITLE
runtime for homebrew games

### DISCUSSION THREAD

**Discussion by PrestigeDevop** [jonathanpeppers/dotnes#35] [General]
hello , thanks for sharing this , I've always wanted to make my own homebrew snes games .
I think I should now c or asm language ... ( there's no way around that) 

but If it cloud use high level framework like [orbital framework ](https://github.com/reignstudios/Orbital-Framework?tab=readme-ov-file) or other [runtime](https://github.com/reignstudios/IL2X) it might be possible to develop tools for or game engine that target these platforms ( or should I say system architectures ) .. 

one day we might be able to use - unity tiny runtime to make an entire game using familiar tools ..or develop wrapper for other libraries .

please keep this project up

**Comment by jonathanpeppers** [comment]
Thanks!

So far this has been a hobby project that I've been mostly doing for fun. If there are others with interest, let me know! I could setup discord about this or something.

I have a few talks coming up about this project:
* Microsoft Build 2024: https://build.microsoft.com/en-US/sessions/0a4b8c8c-bd24-4e77-8df9-0b2e1190c453
* NDC Oslo 2024: https://ndcoslo.com/agenda/running-net-on-the-nes-0wnm/0k9bojgv9zb

**Comment by FallenWyvern** [comment]
> Thanks!
> 
> So far this has been a hobby project that I've been mostly doing for fun. If there are others with interest, let me know! I could setup discord about this or something.
> 
> I have a few talks coming up about this project:
> 
> * Microsoft Build 2024: https://build.microsoft.com/en-US/sessions/0a4b8c8c-bd24-4e77-8df9-0b2e1190c453
> * NDC Oslo 2024: https://ndcoslo.com/agenda/running-net-on-the-nes-0wnm/0k9bojgv9zb

I'm wawtching the MSBUILD talk on it on youtube right now, and it's amazing. If you are looking for interest before starting a discord, you will undoubtedly have a lot of interest! Being able to create readable, clean, NES compatible code in C# is a *dream*.

**Comment by lavedon** [comment]
> Thanks!
> 
> So far this has been a hobby project that I've been mostly doing for fun. If there are others with interest, let me know! I could setup discord about this or something.
> 
> I have a few talks coming up about this project:
> 
> * Microsoft Build 2024: https://build.microsoft.com/en-US/sessions/0a4b8c8c-bd24-4e77-8df9-0b2e1190c453
> * NDC Oslo 2024: https://ndcoslo.com/agenda/running-net-on-the-nes-0wnm/0k9bojgv9zb

Loved the Microsoft Build talk! Congrats!
